test/extended/oauth/requestheaders.go

```diff
@@ -107,6 +105,13 @@ var _ = g.Describe("[Serial] [sig-auth][Feature:OAuthServer] [RequestHeaders] [I
 		o.Expect(err).NotTo(o.HaveOccurred())
 		// clean up after ourselves
 		defer func() {
+			g.By("clean up - wait for stability")
+			ctx, cancel := context.WithTimeout(context.Background(), 20*time.Minute)
+			defer cancel()
+			authenticationRollout := exutil.WaitForOperatorToRollout(ctx, oc.AdminConfigClient(), "authentication")
```

Comment:
I don't think we need to wait here, the next test may not care about the authn operator and so we're just extending the test duration

Reply:
But what if the next test cares? :smile: 

So basically your assumption is that:
- the test doesn't care
- or the test waits for `Progressing=False`?

Reply:
If the test cares then it needs to handle it itself

Reply:
Yes, this is a nice speed increase due to smart logic, but in the state of currently flaky tests, I would prefer dumb logic. 

If we have an attitude to leave the next test to handle the fallout of our tests side effects, it is hard to identify who left the mess at the end.

Reply:
Like I said, I want this code removed.

Reply:
hm kay